.semaphore/semaphore.yml

```diff
@@ -93,9 +99,7 @@ blocks:
     jobs:
     - name: "make build / ci / static-checks"
       commands:
-      - make build
```

Comment:
Probably should change the name of this job to just `make ci`

Reply:
I was thinking it might be nicer to switch away from putting all of the targets into `ci` because when all the commands are tucked inside a single step, its harder to see at a glance which step failed in the semaphore UI. 

I think that's why this one was split out like this, although obviously it was only done halfway since the `ci` target still exists. 

Don't think we need to do anything to this PR, but maybe in the future I'll submit something that breaks out the `ci` targets in semaphore.yaml if you think that's the right way.